test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -798,7 +812,17 @@ private void wipeRollupJobs() throws IOException {
     protected void refreshAllIndices() throws IOException {
         boolean includeHidden = minimumNodeVersion().onOrAfter(Version.V_7_7_0);
         Request refreshRequest = new Request("POST", "/_refresh");
-        refreshRequest.addParameter("expand_wildcards", "open,closed" + (includeHidden ? ",hidden" : ""));
+        refreshRequest.addParameter("expand_wildcards", "open" + (includeHidden ? ",hidden" : ""));
```

Comment:
why did closed get dropped here?

Reply:
Trying to refresh a closed index appears to cause an exception, but that only came up after I switched a test which involves closing an index to use this instead of a direct `POST _refresh` call (to take advantage of the deprecation warning handling). This method is only used in a few tests, and I think specifying `closed` was always wrong, it just never mattered before.